https://issues.redhat.com/browse/ACM-27026 (#8476)
open-cluster-management/rhacm-docs: release_notes/known_issues_virtualization.adoc

{acm-short} 2.15 Virtualization with {ocp-virt} 4.20 is not supported. You need to use {ocp-virt} 4.20.1 or later with {acm-short} 2.15.

== Search result for only virtual machine resources are displayed

> After you enable fine-grained RBAC and attempt to search resource, only virtual machine resources in the `kubevirt.io` `ClusterRoles` resources are displayed. To workaround this issue, disable fine-grained RBAC for virtualization. Complete the following steps:

Reviewer: ```suggestion
After you enable fine-grained RBAC and attempt to search resources, only virtual machine resources in the `kubevirt.io` `ClusterRoles` are displayed and not other resources. To workaround this issue, disable the `fine-grained-rbac-preview` feature. Complete the following steps:
```

Reviewer: I changed a bit because at first it didn't sound like undesired behavior, but it seems like the user wants to see beyond the fine grain feature in their search.
